PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchViewModel.swift

```diff
@@ -11,62 +11,104 @@ enum SearchScope: String, CaseIterable {
 }
 
 class SearchViewModel: ObservableObject {
-
+    static let recentSearchesKey = "SearchViewModel.recentSearches"
     private var subscriptions: [AnyCancellable] = []
     private var networkPathMonitor: NetworkPathMonitor
     private var user: User
+    private var userDefaults: UserDefaults
     private var isOffline: Bool {
         networkPathMonitor.currentNetworkPath.status == .unsatisfied
     }
+
     private var isPremium: Bool {
         return user.status == .premium
     }
+
     private var selectedScope: SearchScope = .saves {
         didSet {
             self.emptyState = isPremium ? premiumEmptyState(for: selectedScope) : freeEmptyState(for: selectedScope)
         }
     }
 
+    @Published
+    var emptyState: EmptyStateViewModel?
+
+    @Published
+    var showRecentSearches: Bool?
+
     var scopeTitles: [String] {
         SearchScope.allCases.map { $0.rawValue }
     }
 
-    init(networkPathMonitor: NetworkPathMonitor, user: User) {
+    var recentSearches: [String] {
```

Comment:
Can we use `@AppStorage` here?

Reply:
I tried at first, but unfortunately `@AppStorage` does not support an array of strings. We can add an extension as a workaround. (https://developer.apple.com/forums/thread/708933) But I decided to use `UserDefaults` since I feel like it intentionally works with an array compared to `AppStorage`. 

Do you see additional benefit in using @AppStorage and using the extension instead of `UserDefaults`?

Reply:
Nah, if it's more work than free "out-of-the-box" then let's leave it as-is!